packages/amplify-category-geo/src/service-walkthroughs/deviceLocationTrackingWalkthrough.ts

```diff
@@ -111,3 +114,129 @@ export const deviceLocationTrackerAccessWalkthrough = async (
   updatedParameters.roleAndGroupPermissionsMap = selectedGroupPermissions;
   return updatedParameters;
 };
+
+/**
+ * deviceLocationTrackerAdvancedWalkthrough
+ */
+export const deviceLocationTrackerAdvancedWalkthrough = async (
+  context: $TSContext,
+  parameters: Partial<DeviceLocationTrackingParameters>,
+): Promise<Partial<DeviceLocationTrackingParameters>> => {
+  let updatedParameters = { ...parameters };
+  updatedParameters.positionFiltering = deviceLocationTrackingPositionFilteringTypes['Time-based'];
+  const selectedAdvancedSetting = await prompter.pick<'one', string>(
+    `Here are the default advanced settings. Select a setting to edit or continue (Use arrow keys)`,
+    [...Object.values(deviceLocationTrackingAdvancedSettings), 'Continue'],
+    { returnSize: 'one' },
+  );
+  switch (selectedAdvancedSetting) {
+    // Grant users access to devices other than their own
+    case deviceLocationTrackingAdvancedSettings.grantOtherAccess:
+      updatedParameters = merge(updatedParameters, await deviceLocationTrackerOtherAccessWalkthrough(updatedParameters));
+      break;
+    // Link Geofence Collection Question
+    case deviceLocationTrackingAdvancedSettings.linkGeofenceCollection:
+      updatedParameters = merge(updatedParameters, await deviceLocationTrackerGeofenceLinkingWalkthrough(updatedParameters));
+      break;
+    // KMS Settings
+    // case deviceLocationTrackingAdvancedSettings.addKMSSettings:
+    //   updatedParameters = merge(updatedParameters, await deviceLocationTrackerKMSSettingsWalkthrough(context, updatedParameters));
+    //   break;
```

Comment:
Instead of commenting this code, I'd rather see it removed and added in another PR when it's needed.